gapis/server/server.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/google/gapid/core/app/benchmark"
 	"github.com/google/gapid/core/event/task"
 	"github.com/google/gapid/core/log"
+	dev "github.com/google/gapid/core/os/device"
```

Comment:
../../gapis/server/server.go:32: imported and not used: "github.com/google/gapid/core/os/device"
../../gapis/server/server.go:135: device.Android undefined (type bind.Device has no field or method Android)